frontend/app-development/hooks/mutations/useEditAppPolicyMutation.ts

```diff
@@ -0,0 +1,22 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useServicesContext } from 'app-shared/contexts/ServicesContext';
+import { PolicyBackendType } from '@altinn/policy-editor/src/types';
+import { QueryKey } from 'app-shared/types/QueryKey';
+
+/**
+ * Mutation to edit an existing policy in an app.
+ *
+ * @param org the organisation of the user
+ * @param app the app the user is in
+ */
+export const useEditAppPolicyMutation = (org: string, app: string) => {
```

Comment:
What do you think about removing the edit form the name, since we already have mutation within the name? :)
```suggestion
export const useAppPolicyMutation = (org: string, app: string) => {
```